README.md

```diff
@@ -0,0 +1,11 @@
+Goldstone is the first purpose-built monitoring, management and analytics platform for operating OpenStack clouds. The Goldstone server allows OpenStack cloud operators to:
+
+* Discover and visualize the OpenStack cloud configuration and logs
+* Report on key events and metrics at all layers of the cloud platform
+
+Future plans including adding the ability to:
+
+* Analyze and diagnose common issues
+* Automate common cloud management and maintenance tasks
+
+Documentation is maintained [HERE](http://goldstone-server.readthedocs.org/en/latest/)
```

Comment:
yes, we had a problem with doc generation that went unnoticed a while ago.  you could look at /en/docker_celery I think.